### DISCUSSION TITLE
Deploying nodejs application locally and in cloud using mrsk

### DISCUSSION THREAD

**Discussion by Arvind644** [basecamp/kamal#194] [General]
Hey, I am looking for alternative to deploy NodeJS application locally and in azure vm and I found about mrsk. I am thinking about deploying the application using mrsk in both environment and write an article to contribute to its docs.

**Comment by huksley** [comment]
Check this out https://medium.com/@ruslanfg/deploying-nextjs-using-mrsk-819aa9eb83ba

**Reply by Arvind644** [reply]
@huksley thanks a lot for sharing your article, I have deployed a sample application to azure vm using mrsk. I omitted some configurations but it worked fine.
